istack-commons/tools/pom.xml

```diff
@@ -81,9 +81,10 @@
                                 <configuration>
                                     <source>9</source>
                                     <target>9</target>
-                                    <includes>
-                                        <include>module-info.java</include>
-                                    </includes>
+                                    <compilerArgs>
+                                        <arg>--add-reads</arg>
+                                        <arg>com.sun.istack.tools=ALL-UNNAMED</arg>
+                                    </compilerArgs>
```

Comment:
I don't like it too, but it's needed to avoid
```
[ERROR] /data/metro/jaxb-istack-commons/istack-commons/tools/src14/com/sun/istack/tools/ProtectedTask.java:[14,24] package org.apache.tools.ant is not visible
 (package org.apache.tools.ant is declared in the unnamed module, but module org.apache.tools.ant does not read it)
```